layouts/shortcodes/civo/OWNERS

```diff
@@ -0,0 +1 @@
+approvers:
```

Comment:
This cannot be empty

Reply:
Yeah I agree, since @shivaylamba created the issue, i suppose maybe he can be the owner. Who else can be the owner for that distro?